### PR TITLE
Create functional for update burst parameter in meters

### PR DESCRIPTION
**Pull request by Levitan**
Add API method for Update burst parameter in meters. Issue #1787

---
This change is [Reviewable](https://reviewable.io/reviews/telstra/open-kilda/1873)

### CODE REVIEW

File: services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java [hunk 1]
```diff
@@ -523,6 +529,26 @@ private void handleReadRequest(String flowId, CommandMessage message, Tuple tupl
...
     }
 
+    private void handleMeterModeRequest(CommandMessage inMessage, Tuple tuple, final String flowId) {
+        FlowPair flowPair = flowService.getFlowPair(flowId)
+                .orElseThrow(() -> new MessageException(inMessage.getCorrelationId(), System.currentTimeMillis(),
+                        ErrorType.NOT_FOUND, "Can not get flow", String.format("Flow %s not found", flowId)));
+
+        SwitchId switchId = flowPair.getForward().getSrcSwitch().getSwitchId();
+        long bandwidth = flowPair.getForward().getBandwidth();
+        Integer meterId = flowPair.getForward().getMeterId();
```
Comment (rtretyak) [inline services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java]: Shouldn't we update meter on the dst switch, too?

Comment (niksv) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java]: Roman is right, We need update meter on last switch too. 
flowPair.getReverse().getSrcSwitch()

Comment (Levitan) [reply services/wfm/src/main/java/org/openkilda/wfm/topology/flow/bolts/CrudBolt.java]: Update

File: services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java [hunk 2]
```diff
@@ -806,21 +864,34 @@ private boolean isCentecSwitch(IOFSwitch sw) {
         return StringUtils.contains(sw.getSwitchDescription().getManufacturerDescription(), "Centec");
     }
 
-    private void buildAndInstallMeter(IOFSwitch sw, Set<OFMeterFlags> flags, long bandwidth, long burstSize,
-                                      long meterId) throws OfInstallException {
-        logger.info("Installing meter {} on switch {} with bandwidth {}", meterId, sw.getId(), bandwidth);
+    private void buildAndInstallMeter(OFMeterMod meterMod, IOFSwitch sw, long bandwidth) throws OfInstallException {
+        logger.info("Installing meter {} on switch {} with bandwidth {}", meterMod.getMeterId(), sw.getId(), bandwidth);
 
+        pushFlow(sw, "--InstallMeter--", meterMod);
```
Comment (niksv) [inline services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java]: I meant that buildAndInstallMeter() and buildAndModifyMeter() must use buildMeterMode() inside themselfes. To do not duplicate meterMode creation, but also do not lose incapsulation of buildAndInstallMeter() and buildAndModifyMeter() methods.

Something like 

```
private void buildAndInstallMeter(IOFSwitch sw, Set<OFMeterFlags> flags, long bandwidth,
     long burstSize, long meterId) throws OfInstallException {	        
***
meterMode = buildMeterMode(***)
pushFlow(sw, "--InstallMeter--", meterMod);


```

**Pull request merged**